examples/official-storybook/stories/__snapshots__/addon-knobs.stories.storyshot

```diff
@@ -19,7 +19,7 @@ exports[`Storyshots Addons|Knobs.withKnobs tweaks static values 1`] = `
     My name is Storyteller, I'm 70 years old, and my favorite fruit is apple.
   </p>
   <p>
-    My birthday is: January 20, 2017
+    My birthday is: January 19, 2017
```

Comment:
@tmeasday looks like your fix don't work in some cases

Reply:
@abrahamgnz just for reference, which timezone are you in?

Reply:
This is puzzling ;)

Reply:
CST @Hypnosphi

Reply:
@abrahamgnz it would be interesting to know what the following output in the node REPL for you:

```js
new Date('Jan 20 2017 GMT+0').toLocaleDateString('en-US', dateOptions)
```

Reply:
I guess looking at the above, it's possible that it would come out as Jan 19. I wonder what time `new Date` picks by default.

Reply:
![image](https://user-images.githubusercontent.com/1657148/34659651-c0fa66fa-f400-11e7-838e-b173af346afe.png)
@tmeasday

Reply:
new Date picks :
2018-01-08T05:16:33.338Z

Reply:
@abrahamgnz -- I guess I mean what does `new Date('Jan 20 2017 GMT+0')` pick? 

For me its `2017-01-20T00:00:00.000Z`, and it evaluates to "1/20/2017".

Also:

```js
> +new Date('Jan 20 2017 GMT+0')
1484870400000
```

Reply:
I would expect all of the above to be system/TZ independent; unless something is weird on your machine with the `en-US` TZ?

Reply:
```
> new Date('Jan 20 2017 GMT+0')
2017-01-20T00:00:00.000Z
```

Reply:
```
+new Date('Jan 20 2017 GMT+0')
1484870400000
```
@tmeasday

Reply:
@tmeasday looks like we have to add `timeZone: 'UTC'`  to `dateOptions`
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/toLocaleDateString

Reply:
I can't believe no-one in the US ran into this yet... not even the CI servers!

I can't really explain why it was printing out the 19th in the first place for me then (given `new Date('Jan 20 2017')` was midnight 20 Jan in my TZ). But let's lock it down.

Reply:
how?

Reply:
This was probably fixed in #2861 

@abrahamgnz can you please try to update snapshots once again?